bin/resty

```diff
@@ -289,6 +289,11 @@ my $lua_shared_dicts = '';
                 next;
             }
 
+            if ($arg =~ /^--dry-run$/) {
```

Comment:
This is not really a "dry run" in the common sense and thus is confusing. I suggest using the option name `--dump-nginx-conf` instead? Better be explicit for such features.